src/roc-zenodo/getZenodoDeposition.js

```diff
@@ -67,10 +69,22 @@ function validateCreators(creators) {
     if (typeof creator.affiliation !== 'string') {
       throw new TypeError('creator must have an affiliation');
     }
-    toReturn.push({
-      name: creatorName,
-      affiliation: creator.affiliation,
-    });
+    if ('orcid' in creator) {
```

Comment:
Please create an initial object without the `orcid` property, and then add it if it was provided. Something like

```js
const creatorToReturn = { name: 'xxx', affiliation: 'yyy' }
if (creator.orcid) {
  if (okOrcid(creator.orcid) {
    creatorToReturn.orcid = creator.orcid
  } else {
    throw ...
  }
}
toReturn.push(creatorToReturn)
```